test/integration/scan_test.go

```diff
@@ -463,7 +463,7 @@ func executeScanAssertions(t *testing.T, projectID, scanID string, tags map[stri
 }
 
 func createScan(t *testing.T, source string, tags map[string]string) (string, string) {
-	return executeCreateScan(t, getCreateArgs(source, tags, "sast , sca , iac-security , api-security   "))
+	return executeCreateScan(t, getCreateArgs(source, tags, "sast , sca , iac-security , api-security  "))
```

Comment:
we want to add container here. BTW, there is a white space in api-secuirty. we shouldn't have (unless we want to check how we are handling this)

Reply:
maybe I will add the containers here after checking the FF
because if we add it always, it will appear in the result report with 0 results
what do you say?